samples/widom_insertion.py

```diff
@@ -113,6 +113,10 @@
 print(widom.get_status())
 system.setup_type_map([0, 1, 2])
 
+
+# Set the hidden particle type to the lowest possible number to speed up the simulation
+widom.set_non_interacting_type(len(types))
```

Comment:
`max(types)` would be safer here: there is no guarantee that particle types are numbered sequentially starting from 0